x/ccv/provider/keeper/relay.go

```diff
@@ -300,24 +309,65 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 // EndBlockCIS contains the EndBlock logic needed for
 // the Consumer Chain Removal sub-protocol
 func (k Keeper) EndBlockCCR(ctx sdk.Context) {
-	currentTime := uint64(ctx.BlockTime().UnixNano())
+	currentTime := ctx.BlockTime()
+	currentTimeUint64 := uint64(currentTime.UnixNano())
```

Comment:
Why do you need the Uint64 version here? Can't you compare Go time stuff?

Reply:
see https://github.com/cosmos/interchain-security/pull/422#discussion_r1014468716